src/ILVerify/src/ILImporter.Verify.cs

```diff
@@ -1276,7 +1266,21 @@ void ImportStoreIndirect(int token)
 
         void ImportStoreIndirect(TypeDesc type)
         {
-            throw new NotImplementedException($"{nameof(ImportStoreIndirect)} not implemented");
+            ClearPendingPrefix(Prefix.Unaligned);
+            ClearPendingPrefix(Prefix.Volatile);
+
+            var value = Pop();
+            var address = Pop();
+
+            Check(!address.IsReadOnly, VerifierError.ReadOnlyIllegalWrite);
+
+            CheckIsByRef(address);
+            if (type != null)
+                CheckIsAssignablePointer(type, address.Type);
+            else
+                CheckIsObjRef(address.Type);
+
+            CheckIsAssignable(value, address.DereferenceByRef());
```

Comment:
@mmayr-at Docu on that is in the readme file 😉 https://github.com/dotnet/corert/blob/master/src/ILVerify/README.md